modules/support-submitting-a-case.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * support/getting-support.adoc
+
+[id="support-submitting-a-case_{context}"]
+= Submitting a support case
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* A Red Hat Customer Portal account.
+* A Red Hat standard or premium subscription.
+
+.Procedure
+
+. Login to the link:http://access.redhat.com[Red Hat Customer Portal] and select *SUPPORT CASES* -> *Open a case*.
+
+. Select the appropriate category for your issue (such as 'Defect / Bug'), product ('{product-title}') and product version ('{product-version}', if this is not already autofilled).
+
+. Review the list of suggested Red Hat Knowledgebase solutions for a potential match against the problem that is being reported. If the suggested articles do not address the issue, click *Continue*.
+
+. Enter a concise but descriptive problem summary and further details about the symptoms being experienced, as well as your expectations.
+
+. Review the updated list of suggested Red Hat Knowledgebase solutions for a potential match against the problem that is being reported. The list is refined as you provide more information during the case creation process. If the suggested articles do not address the issue, click *Continue*.
+
+. Ensure that the account information presented is as expected and if not, amend accordingly.
+
+. Check that the autofilled OpenShift Cluster ID is correct. If it is not, manually obtain your cluster ID:
++
+* To manually obtain your cluster ID using the {product-title} web console:
+.. Navigate to *Home* -> *Dashboards* -> *Overview*.
+.. The value is available in the *Cluster ID* field of the *Details* section.
```

Comment:
s/The value is available in/Find the value in/

Reply:
Resolved.